css/css-flexbox-exercise-1.css

```diff
@@ -0,0 +1,119 @@
+body {
+    margin: 0% 5%;
+    
+}
+
+h1 {
+    font-size: 24px;
+    font-weight: 600;
+    padding: 15px 15px;
+    margin-top: 10px;
+    border-top: 1px solid #000;
+    border-bottom: 3px solid #48B99D;
+}
+
+.shopping_list {
+    margin-top: 5px;
+    display: flex; 
+    justify-content: space-between;
+    border-top: 3px solid #48B99D;
+    column-gap: 5px; 
+    
+    
+}
+
+.box {
+    display: flex;
+    flex-direction: column; 
+    align-items: flex-start;
+    justify-content: center;
+    margin-top:10px;
+    border: 1px solid #000;
+    border-radius: 5px;
+    flex-basis: 100%;
+}
+
+.background-img-1 {
+    display: flex;
+    align-items: center;
+    background-image: url("../img/warzywa.png");
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 150px;
+}
+
+.background-img-2 {
```

Comment:
masz sporą redundancje w stylach do klas `.background-img-1 .background-img-2 i .background-img-3` imo można by spokojnie im stworzyć 1 klase i zrobić to np tak
```css
.shopping-item__header {
    display: flex;
    align-items: center;
    background-repeat: no-repeat;
    width: 100%;
    height: 150px;
}
```
i potem wykorzystać twoje klasy do backgrounda zeby tylko ustawić sam background-image
```css
.background-img-1 {
     background-image: url("../img/warzywa.png");
}

/* albo zrobić bardziej semantyczne klasy np */
.shopping-item__header_fruits

Reply:
Redundancja -> Nowe słowo do słownika ;)